lighthouse-logger/index.js

```diff
@@ -128,6 +128,72 @@ class Log {
     marky.stop(id);
   }
 
+  /* eslint-disable no-invalid-this */
+  static timeDecorate(originalFn, opts) {
+    return function timeDecoratedFn(...args) {
+      const timeStartLogLevel = opts.timeStartLogLevel || 'log';
```

Comment:
Attempting to maintain the same feature set as before, with how messages are only logged w/ the proper verbosity level.